<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menug">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action_reload_config_file"/>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menug"/>
  </widget>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="exit"/>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action_reload_config_file">
   <property name="icon">
    <iconset theme="reload"/>
   </property>
   <property name="text">
    <string>Recharger le fichier de configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
